README.md

```diff
@@ -137,6 +137,58 @@ print('\nDeleting index:')
 print(response)
 ```
 
+## Using IAM credentials for authentication
+
+Refer the AWS documentation regarding usage of IAM credentials to sign requests to OpenSearch APIs - [Signing HTTP requests to Amazon OpenSearch Service.](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/request-signing.html#request-signing-python)
+
+Opensearch-py client library also provides an in-house IAM based authentication feature, `AWSV4SignerAuth` that will help users to connect to their opensearch clusters by making use of IAM roles.
+
+#### Pre-requisites to use `AWSV4SignerAuth`
+ - Python version 3.6 or above,
+ - Install [botocore](https://pypi.org/project/botocore/) using pip
+
+   `pip install botocore`
+
+Here is the sample code that uses `AWSV4SignerAuth` -
+
+```python
+from opensearchpy import OpenSearch, RequestsHttpConnection, AWSV4SignerAuth
+import boto3
+
+host = '' # cluster endpoint
```

Comment:
maybe give example of host definition here, this can save setup time for customers. For instance, seems that port mustn't be in the endport URI, customer may also put endpoint with the protocol - it's not clear whether this ok or not

Reply:
sure, updated.